test/protocol/eval_test.rb

```diff
@@ -23,5 +23,14 @@ def test_eval_evaluates_arithmetic_expressions
         req_terminate_debuggee
       end
     end
+
+    def test_eval_workaround
+      run_protocol_scenario PROGRAM do
+        req_add_breakpoint 5
+        req_continue
+        assert_repl_result({value: '', type: 'String'}, ',eval 1+1')
```

Comment:
Unfortunately the first approach doesn't work because `res` doesn't actually contain the outputted value:

```ruby
{:type=>"response", :command=>"evaluate", :request_seq=>10, :success=>true, :message=>"Success", :body=>{:result=>"", :variablesReference=>0}, :seq=>13}
```